### ISSUE TITLE
Broadcasting

### ISSUE THREAD

**Issue by davidweichiang** [bdusell/semiring-einsum#10]
torch.einsum now supports broadcasting: https://github.com/pytorch/pytorch/issues/48420

**Comment by bdusell** [comment]
Yes, this would be a good feature to add, and I don't think it would be too tough to implement.

**Comment by davidweichiang** [comment]
Maybe just call torch.expand on the arguments beforehand?

**Comment by davidweichiang** [comment]
Or is it possible that it just works already, but the dimension checks are too strict?
